tests/conftest.py

```
@@ -0,0 +1,21 @@
+"""
+
+import sys
+import types
+
+# Shim the top-level gliner package so that ``import gliner.config`` does NOT
+# trigger ``gliner/__init__.py`` (which imports torch via model.py).  The shim
+# only needs to set __path__ so that sub-module resolution works.
+# Only apply the shim when torch is NOT available; when torch is present,
+# let the real gliner package load normally so GLiNER/GLiNERConfig are accessible.
+try:
+    import torch  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)